lib/rdkafka/helpers/oauth.rb

```diff
@@ -0,0 +1,45 @@
+module Rdkafka
+  module Helpers
+
+    class OAuth
+
+      # Set the OAuthBearer token
```

Comment:
@mensfeld  Do you want the docstring on each of the methods in admin, consumer, producer?

Reply:
I wonder why not generalize it once in a helper and include